src/main/java/pl/sda/library/domain/model/exception/AuthorAlreadyExistException.java

```diff
@@ -0,0 +1,7 @@
+package pl.sda.library.domain.model.exception;
+
+public class AuthorAlreadyExistException extends ObjectAlreadyExistException {
+    public AuthorAlreadyExistException() {
```

Comment:
przekazuj tutaj id autora

Reply:
albo nawet lepiej - caly obiekt autora

będziesz wtedy dokladnie widzial w logach jakiego autora nie udalo sie dodac (bedzie widoczne jego imie i nazwisko)

Reply:
np. tak
public AuthorAlreadyExistException(Author author) {
        super("Author " + author.getFirstName() + " " + author.getLastName() + " already exists in the database");

Reply:
Moze:
"Author already exist: " + author.toString()

Reply:
lepiej ;)